R/reduce.R

```diff
@@ -139,7 +139,7 @@ seq_len2 <- function(start, end) {
 #'
 #' `accumulate` applies a function recursively over a list from the left, while
 #' `accumulate_right` applies the function from the right. Unlike `reduce`
-#' both functions keep the intermediate results.
+#' both functions keep the intermediate results and the names of `.x`.
```

Comment:
Rather than adding to this sentence (where it doesn't quite belong). I think it would be better to add a `@return` section which describes the result value (i.e. A vector the same length of `.x` with the same names as `.x`)